packages/ibm-products/src/components/Toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import cx from 'classnames';
+import { bool, node, string } from 'prop-types';
+
+import React, {
+  PropsWithChildren,
+  ReactNode,
+  createContext,
+  forwardRef,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+// cspell:words Focusable focusable
+import { getFocusableElements as _getFocusableElements } from '../../global/js/utils/getFocusableElements';
+import { pkg } from '../../settings';
+
+const { checkComponentEnabled, prefix } = pkg;
+const blockClass = `${prefix}--toolbar`;
+
+interface ToolbarProps {
+  /** Provide the content of the `ToolbarGroup` */
+  children: ReactNode;
```

Comment:
Not necessary to include `children` since you've included the generic type `PropsWithChildren`

Reply:
removed the children